src/index.js

```diff
@@ -108,6 +108,12 @@ class Purgecss {
     getCssContents(cssOptions: Array<any>, cssSelectors: Set<string>): Array<ResultPurge> {
         const sources = []
 
+        // resolve any globs and flatten again
+        cssOptions = cssOptions.map(option => {
+            return typeof option === 'string' ? glob.sync(option) : option
+        })
+        cssOptions = [].concat.apply([], cssOptions)
```

Comment:
could we maybe use the native `Array.prototype.flatten()` method instead of that hack?

Reply:
@jsnanigans I was doing that originally, but it's not in node yet...

Reply:
huh strange but OK then

Reply:
I really hope they don't call it `smoosh`